plugins/contiv/containeridx/persist_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package containeridx
+
+import (
+	"github.com/contiv/vpp/mock/broker"
+	"github.com/contiv/vpp/plugins/contiv/containeridx/model"
+	"github.com/ligato/cn-infra/core"
+	"github.com/ligato/cn-infra/logging/logrus"
+	"github.com/onsi/gomega"
```

Comment:
Could use `. "github.com/onsi/gomega` for more readable code, without `gomega.` prefixes all over the place.